types/jasmine-reporters/index.d.ts

```diff
@@ -0,0 +1,81 @@
+// Type definitions for Jasmine Reporters 2.5
+// Project: https://github.com/larrymyers/jasmine-reporters
+// Definitions by: Adam Kwiatek <https://github.com/akwiatek>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import jasmine = require('jasmine');
+
+// tslint:disable-next-line:no-unnecessary-class
+export class AppVeyorReporter implements jasmine.CustomReporter {
+    constructor(options?: AppVeyorReporter.ConstructorOptions);
+}
+
+export namespace AppVeyorReporter {
+    interface ConstructorOptions {
+        batchSize: number;
+        color: boolean;
+        verbosity: 0 | 1 | 2;
```

Comment:
all of thos are marked as optional in constructor jsdoc:
https://github.com/larrymyers/jasmine-reporters/blob/master/src/appveyor_reporter.js#L32-L35

Reply:
Done.